scipy/stats/_multivariate.py

```diff
@@ -3328,6 +3328,12 @@ class special_ortho_group_gen(multi_rv_generic):
     >>> rv = special_ortho_group(5)
     >>> # Frozen object with the same methods but holding the
     >>> # dimension parameter fixed.
+    >>> rv.rvs(random_state=1234)
```

Comment:
I would say it's good to add this, but there are other issues here that I don't want to get into (should use `np.random.default_rng()`, comments on the previous two lines should not be there). 
IIRC, all multivariate distributions need this same sort of treatment. Let's do that to all of them in a separate PR and limit the scope here to just adding the frozen ortho group class. Sound good?

Reply:
Yeah, that sounds good. Should I remove the previous two comment lines?